pkg/timeutil/timeutil.go

```diff
@@ -26,3 +42,22 @@ func LastWeekday(weekday time.Weekday) time.Time {
 
 	return date
 }
+
+// StringToWeekday maps a string to a time.Weekday.
+func StringToWeekday(day string) (time.Weekday, error) {
+	timeWeekday, ok := weekday[strings.ToLower(day)]
+	if !ok {
+		return time.Sunday, fmt.Errorf("timeutil: Invalid weekday '%s' given", day)
+	}
+
+	return timeWeekday, nil
+}
+
+// FormatDuration prints the time using the given format.
+func FormatDuration(duration time.Duration, timeFormat string) string {
+	if timeFormat == "decimal" {
```

Comment:
Something to consider here: timeFormat could be it's own type, defined in the "types" package. This might not be something you've come across yet so i'll do my best to explain:

In other places in the application you may wish to check if the timeFormat is decimal (or maybe other things further down the line)

In go, you can declare types like so:

```go
type TimeFormat string
```

and this could maybe go in the types package. This says "TimeFormat is a new type but it's a string really"

then what you can do, is define a constant as that type (again in the types package), e.g.

```go
const (
    Decimal TimeFormat = "decimal"
)
```

then your above code would take timeFormat as `types.TimeFormat`, and finally, you can compare the timeFormat given to the constant defined in the types package, like so:

```go
if timeFormat == types.Decimal 
```

Essentially what you're doing is defining a constant of a specific type, which then allows you to check it further down the line.

(if this is explained horribly I apologise :D)